src/components/Header/logo.tsx

```diff
@@ -0,0 +1,20 @@
+import { HStack, Link, Image } from "@chakra-ui/react";
+
+export default function Logo() {
+  return (
+    <>
+      {/* Stacking */}
+      <HStack spacing={8} align={'center'}>
+        {/* Logo */}
+        <Link href={'/main'} _hover={{ opacity: '50%' }}>
```

Comment:
@kevinjung13 main does not exist should be `/` for index
Same here, make sure to use next/link

Reply:
Done, and I now understand the difference next/link makes. Thanks for sharing!